File: Backend/src/main/java/com/jfrb/POCSucripcion/model/SuscripcionStreaming.java
package com.jfrb.POCSucripcion.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "SUSCRIPCION_STREAMING")
public class SuscripcionStreaming {

    @Id
    private int id;

    private LocalDateTime fechaInicio;
    private boolean activa;
    private int dispositivosSimultaneos;
    private double costoMensual;
    private String plataforma;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "CODIGO_USUARIO", nullable = false)
    @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
    private Usuario usuario;
}

File: Cliente1 VS/POCSuscripcionCliente/Models/SuscripcionStreaming.cs
using System;

namespace POCSuscripcionCliente.Models
{
    public class SuscripcionStreaming
    {
        public int id { get; set; }
        public DateTime fechaInicio { get; set; }
        public bool activa { get; set; }
        public int dispositivosSimultaneos { get; set; }
        public double costoMensual { get; set; }
        public string plataforma { get; set; }
        public Usuario usuario { get; set; }

        public string EstadoTexto
        {
            get
            {
                return activa ? "Activa" : "Inactiva";
            }
        }

        public string UsuarioNombre
        {
            get
            {
                return usuario != null ? usuario.nombre : "";
            }
        }

        public int CodigoUsuario
        {
            get
            {
                return usuario != null ? usuario.codigo : 0;
            }
        }
    }
}